<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-D2GVCL39" name="invocation.Invocation" tests="5" failures="1" timestamp="2022-10-12T20:21:05 IST" time="0.007" errors="0">
  <testcase name="verifyUserCanLoginToApplicationWithValidCredentials" time="0.001" classname="invocation.Invocation">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at invocation.Invocation.verifyUserCanLoginToApplicationWithValidCredentials(Invocation.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyUserCanLoginToApplicationWithValidCredentials -->
  <system-out/>
  <testcase name="verifyUserCanLoginToApplicationWithValidCredentials" time="0.001" classname="invocation.Invocation"/>
  <system-out/>
  <testcase name="verifyUserCanLoginToApplicationWithValidCredentials" time="0.002" classname="invocation.Invocation"/>
  <system-out/>
  <testcase name="verifyUserCanLoginToApplicationWithValidCredentials" time="0.001" classname="invocation.Invocation"/>
  <system-out/>
  <testcase name="verifyUserCanLoginToApplicationWithValidCredentials" time="0.002" classname="invocation.Invocation"/>
  <system-out/>
</testsuite> <!-- invocation.Invocation -->
